client/src/views/CellView.tsx

```diff
@@ -101,20 +122,62 @@ const CellView = () => {
     });
   }, [Alert, Confirm, brute, t]);
 
+  useEffect(
+    () => {
+      if (!user || !brute || user.id !== brute.userId) return () => {};
+      let touchstartX = 0;
+      let touchendX = 0;
+
+      const checkSwipe = () => {
+        if (touchendX > touchstartX && touchendX - touchstartX > 120) {
+          switchBrute(-1);
+        }
+        if (touchendX < touchstartX && touchstartX - touchendX > 120) {
+          switchBrute(1);
+        }
+      };
+      const handlerStart = (e : TouchEvent) => {
+        touchstartX = e.changedTouches[0].screenX;
+      };
+      const handlerEnd = (e : TouchEvent) => {
+        touchendX = e.changedTouches[0].screenX;
+        checkSwipe();
+      };
+      document.addEventListener('touchstart', handlerStart);
+      document.addEventListener('touchend', handlerEnd);
+
+      return () => {
+        document.removeEventListener('touchstart', handlerStart);
+        document.removeEventListener('touchend', handlerEnd);
+      };
+    },
+
+    [brute, navigate, switchBrute, user]
+  );
+
+  const ArrowPrevious = (<Fab size={smallScreen ? 'small' : 'medium'} onClick={() => switchBrute(-1)} sx={{ position: 'absolute', backgroundColor: 'primary.main', left: '25px', top: '50%', cursor: 'pointer', margin: 'auto', }}><NavigateBeforeIcon sx={{ color: 'secondary.main', cursor: 'pointer', margin: 'auto' }} /></Fab>);
```

Comment:
All variables should be in camelcase (`previousBruteArrow`).

You can also refactor the condition in the variable directly

`const previousBruteArrow = ownsBrute && (...)`